src/group-configurations/group-configuration-container/index.jsx

```diff
@@ -8,22 +8,36 @@ import {
   Hyperlink,
   Icon,
   IconButtonWithTooltip,
+  useToggle,
 } from '@edx/paragon';
 import {
   DeleteOutline as DeleteOutlineIcon,
   EditOutline as EditOutlineIcon,
 } from '@edx/paragon/icons';
 
+import DeleteModal from '../../generic/delete-modal/DeleteModal';
+import ContentGroupContainer from '../content-groups-section/ContentGroupContainer';
 import ExperimentGroupStack from './ExperimentGroupStack';
 import TitleButton from './TitleButton';
 import UsageList from './UsageList';
 import messages from './messages';
 
-const GroupConfigurationContainer = ({ group, isExperiment, readOnly }) => {
+const GroupConfigurationContainer = ({
```

Comment:
[optional]: I would split this component into several smaller ones. For example: `ConfigurationCardHeader`, `ConfigurationCardContent`.

Reply:
Will be decomposed in the following PR with experiment group